Sources/TuistCache/Utilities/CacheXCFrameworkBuilder.swift

```diff
@@ -15,14 +29,22 @@ public final class CacheXCFrameworkBuilder: CacheArtifactBuilding {
 
     /// Initializes the builder.
     /// - Parameter xcodeBuildController: Xcode build controller instance to run xcodebuild commands.
-    public init(xcodeBuildController: XcodeBuildControlling) {
+    /// - Parameter cacheOutputType: Output type of xcframework (device and/or simulator)
+    public init(
+        xcodeBuildController: XcodeBuildControlling,
+        cacheOutputType: CacheOutputType
```

Comment:
This is an XCFrameworkBuilder, it should just take a `CacheXCFrameworkDestination`, the type is xcframework by definition